tailwind.config.js

```diff
@@ -39,7 +40,7 @@ export default {
           105: "#ced4da",
           106: "#9E9E9F",
           107: "#1E1E1F",
-          108: "#424245"
+          108: "#424245",
```

Comment:
seems like #424245 is already defined 'black'

Reply:
It was thasnim's change. Anyways, I will fix it.